webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/domain/reports/StatusReportTest.scala

```diff
@@ -297,13 +295,10 @@ class StatusReportTest extends Specification {
 
       "run fast enough to sum" in {
         ComplianceLevel.sum(initData.map( x => ComplianceLevel.compute(x._2)))
-
-        val source = initData.map(x => (x._1, ComplianceLevel.compute(x._2))).map( _._2)
         val t0 = System.nanoTime
 
         for (i <- 1 to 100) {
           val t0_0 = System.nanoTime
-          val result = ComplianceLevel.sum(source)
```

Comment:
these should not be removed. They are used to warm the JVM, plus are not at all related to this ticket

Reply:
Those variable are unused, and really look like dead code

Reply:
@ncharles if they are useful, we need a way to mark them as that. I regularly remove things that lead to warning or looks like dead code. At minima, they need to have: 
- a comment to explain why they are here,
- a silent annotation to remove warning. 
Rudder should build with 0 warning